R/render.R

```diff
@@ -847,9 +847,16 @@ render <- function(input,
       utf8_input <- path.expand(utf8_input)
       output     <- path.expand(output)
 
+      pandoc_args <- output_format$pandoc$args
+
+      # if lua filters are provided, add the command line switch
+      if (!is.null(lua_filters <- output_format$pandoc$lua_filters)) {
+        lua_filters <- pandoc_lua_filter_args(lua_filters)
+      }
+      pandoc_args <- c(pandoc_args, lua_filters)
```

Comment:
Considering https://github.com/rstudio/rmarkdown/issues/1897#issuecomment-692117149, I'd suggest a change so that user-specified lua filters comes after document-builtin filters.

```suggestion
      pandoc_args <- c(lua_filters, pandoc_args)
```

Reply:
Yes I think you're right. I was hesitating on this one because not sure if it is easier to add a filter that modify first the AST or last, but to be sure our filters are working, they should be first run.